src/routes/index.js

```diff
@@ -15,5 +16,6 @@ router.use('/admin-invitations', adminInvitation)
 router.use('/questions', question)
 router.use('/resources-categories', resourcesCategory)
 router.use('/offers', offer)
+router.use('/uploadImage', uploadImage)
```

Comment:
This endpoint allows us to upload an image, but how are we going to store the URL of that image for the user? Let's discuss this, it can be done this way.